--- a/blitzbrowser/src/components/browser-instance.process.ts
+++ b/blitzbrowser/src/components/browser-instance.process.ts
@@ -198,7 +198,8 @@ class BrowserInstanceProcess {
                 '--disable-features=PersistentHistograms', // Prevents generation of BrowserMetrics files on disk
                 `--start-maximized`,
                 `--proxy-server=http://127.0.0.1:${this.#proxy_server_port}`,
-                this.#disable_shm ? '--disable-dev-shm-usage' : undefined
+                this.#disable_shm ? '--disable-dev-shm-usage' : undefined,
+                '--no-sandbox',
             ].filter(s => typeof s === 'string'),
             env: {
                 TZ: this.#timezone
